python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -116,7 +116,7 @@ def load_subJobIndex(self):
 
         raw_self = stripProxy(self)
```

Comment:
Makes sense. This wasn't introduced in this PR so perhaps shouldn't be fixed in it.